boutvecma/encoder.py

```diff
@@ -1,7 +1,50 @@
+import os
+from copy import deepcopy
 from easyvvuq.encoders import BaseEncoder
+from boutdata.data import BoutOptions, BoutOptionsFile
+
 
 class BOUTEncoder(BaseEncoder, encoder_name="bout++_encoder"):
```

Comment:
Going off other encoders, this should probably be

```suggestion
class BOUTEncoder(BaseEncoder, encoder_name="bout++"):
```